rpm_spec/core-vm.spec

```diff
@@ -198,6 +198,13 @@ DNF plugin for Qubes specific post-installation actions:
  * refresh applications shortcut list
 %endif
 
+%package thunar
```

Comment:
Base package name in r3.2 is `qubes-core-vm`, so this will result in `qubes-core-vm-thunar` package. This is not what builder-fedora expects. That was renamed to qubes-core-agent in r4.0 (to be consistent with debian and repository name).
So, here better use `%package -n qubes-core-agent-thunar` (and similar for other % macros) to have the same package name everywhere.